out/notebooks/spl2/installer.ts

```diff
@@ -299,7 +309,7 @@ async function installJDK(installDir: string, progressBar: StatusBarItem): Promi
         if (!binJavaPath) {
             // If not found during unpacking, search for bin/java[.exe] and check that file
             let pathSuffix = path.join('bin', 'java');
-            if (true || process.platform === 'win32') {
+            if (process.platform === 'win32') {
```

Comment:
fixed this hardcoding that was leftover from my local testing of windows extraction 🤦

Reply:
Why don't we need to check for `win64` (or potentially other distros) here? Is there something specific to `win32` happening?

Reply:
this branch of the code is to use `java.exe` instead of the `java` binary which is only valid for windows. and it turns out nodejs is weird and even x64 builds of Windows still have a platform value of `win32` https://stackoverflow.com/questions/8683895/how-do-i-determine-the-current-operating-system-with-node-js#answer-8684009 🙃